test/e2e/util.go

```diff
@@ -311,7 +311,13 @@ func createTestingNS(baseName string, c *client.Client) (*api.Namespace, error)
 		Status: api.NamespaceStatus{},
 	}
 	_, err := c.Namespaces().Create(namespaceObj)
-	return namespaceObj, err
+	if err != nil {
+		return namespaceObj, err
```

Comment:
yeah, looks like tons of tests pull out the name from the returned namespace, then check for the error... would rather not sweep all tests in this PR